src/controllers/userController.js

```diff
@@ -0,0 +1,69 @@
+const {
+  getUsers,
+  getUserById,
+  create,
+  remove,
+  update,
+} = require('../services/userService');
+
+const getAll = (req, res) => {
+  res.statusCode = 200;
+  res.send(getUsers());
```

Comment:
Consider wrapping the service call `getUsers` in a try-catch block to handle any potential errors that might occur during the execution.